ansible_base/authentication/utils/claims.py

```diff
@@ -263,21 +266,34 @@ def update_user_claims(user: Optional[AbstractUser], database_authenticator: Aut
 
     # Make the orgs and the teams as necessary ...
     if database_authenticator.create_objects:
-        process_organization_and_team_memberships(results)
+        create_organizations_and_teams(results)
 
-    # We have allowed access so now we need to make the user within the system
-    reconcile_class = getattr(settings, 'ANSIBLE_BASE_AUTHENTICATOR_RECONCILE_MODULE', 'ansible_base.authentication.utils.claims')
-    try:
-        module = __import__(reconcile_class, fromlist=['ReconcileUser'])
-        klass = getattr(module, 'ReconcileUser')
-        klass.reconcile_user_claims(user, authenticator_user)
-    except Exception as e:
-        logger.error(f"Failed to reconcile user attributes! {e}")
+    if reconcile_user_class := load_reconcile_user_class():
+        try:
+            # We have allowed access, so now we need to make the user within the system
+            reconcile_user_class.reconcile_user_claims(user, authenticator_user)
+        except Exception as e:
+            logger.exception("Failed to reconcile user claims: %s", e)
 
     return user
 
 
-def process_organization_and_team_memberships(results) -> None:
+def load_reconcile_user_class():
+    module_path = getattr(settings, 'ANSIBLE_BASE_AUTHENTICATOR_RECONCILE_MODULE', 'ansible_base.authentication.utils.claims')
+    try:
+        module = importlib.import_module(module_path)
+    except ImportError:
+        logger.warning("Failed to load module '%s'.", module_path)
+        return None
+
+    try:
+        return getattr(module, 'ReconcileUser')
+    except AttributeError:
+        logger.warning("Failed to load ReconcileUser class in module '%s'.", module_path)
+        return None
```

Comment:
This might actually be nice to make generic in a "lib/utils" file at some point. I think there are other places where we try and load things dynamically just like this. Non-blocking on this PR.